vpr/src/route/partition_tree.h

```diff
@@ -42,6 +55,8 @@ class PartitionTreeNode {
   public:
     /** Nets claimed by this node (intersected by cutline if branch, nets in final region if leaf) */
     std::vector<ParentNetId> nets;
+    /** Nets assigned by the parent of this node (\see DecompNetlistRouter) */
```

Comment:
Is this the right terminology?  Nets assigned by the parent of this node?  It seems like this should be the virtual nets (decomposed nets) claimed by this node -- is that right?  (As some of the virtual nets sent down to this level may wind up being pushed further down into the leaves).  It seems to me that this comment for vnets shouldn't be "Nets assigned by the parent of this node".

Reply:
It means virtual nets assigned by the parent of this node: the router may choose to fully route or further decompose them.